### PR TITLE
Translated files in Italian

### PR DESCRIPTION
**Pull request by LuigiImVector**
## What does this PR do?
Add resource(s) | Improve repo

### Description

I translated the CODE_OF_CONDUCT, CONTRIBUTING and HOWTO respecting #5644.
Sorry if I made one big pull request.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.

### CODE REVIEW

File: HOWTO-it.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+* [Creare una pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+* [Github Hello World](https://guides.github.com/activities/hello-world/)
+* [Youtube - Github Tutorial per Principianti](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [Youtube - Come forkare una Repository Github e Inviare una Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [Youtube - Corso accelerato di Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+
+Non esitare a fare domande; ogni collaboratore iniziato con una prima pull request (PR). Potresti essere il nostro millesimo collaboratore.
+
+Anche se sei un collaboratore esperto in ambito open source, queste sono cose che potrebbero crearti dei problemi. Una volta che hai inviato la tua PR, Github Actions  avvierà un linter, spesso trovando piccoli problemi di spaziatura o di ordinamento alfabetico. Se ottieni una spunta verde, tutto è pronto per una revisione, ma se non è così clicca su "Details" sotto il check che è stato fallito, analizza l'errore, risolvilo e aggiungi un commit alla PR.
```
Comment (davorpa) [inline HOWTO-it.md:15]: It satisfy #5644

File: CONTRIBUTING-it.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    - *Corsi* : Un corso è del materiale gratuito che non è un libro. [Questo è un corso](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-006-introduction-to-algorithms-fall-2011/).
+    - *Tutorial Interattivi* : Un sito interattivo permette all'utente di scrivere codice o comandi e analizzarne il risultato. esempi: [Try Haskell](http://tryhaskell.org), [Try Github](http://try.github.io).
+    - *Podcasts e Screencasts* : Podcasts and screencasts.
+    - *Set di problemi & Programmazione competitiva* : Un sito o software che ti permette di valutare le tue skills da programmatore risolvendo problemi semplici o complessi, con o senza la revisione del codice, con o senza la comparazione del risultato con gli altri utenti.
+
+4. Assicurati di seguire le [linee guida qui sotto](#guidelines) e rispettare la [formattazione Markdown](#formatting) dei file.
+
+5. GitHub Actions avvierà dei test per assicurarsi che le tue liste siano ordinate alfabeticamente e formattate correttamente. Assicurati che i tuoi cambiamenti passino il test.
+
+### Linee guida
```
Comment (davorpa) [inline CONTRIBUTING-it.md:25]: :warning: Marker anchor at line 21 doesn't work :worried:. To be applied:

```markdown
<a name="guidelines"></a>
### Linee guida
```

File: CONTRIBUTING-it.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+    + se reindirizzano a edizioni differenti e tu credi che queste edizioni siano abbastanza diverse tra loro da valere la pena di essere tenute, aggiungi un link separato con una nota riguardante ogni edizione (guarda [Issue #2353](https://github.com/EbookFoundation/free-programming-books/issues/2353) per contribuire alla discussione sulla formattazione.)
+- preferisci gli atomic commits (un commit per aggiunta/modifica/eliminazione) rispetto ai grandi commit. Non c'è bisogno di raggruppare i commit per inviarli in una sola PR. (Non applichiamo mai questa regola, è solo per comodità dei moderatori)
+- se il libro è più vecchio, includi la data di pubblicazione assieme al titolo.
+- includi il nome o i nomi degli autori se è il caso. Puoi accorciare il nome degli autori con "et al."
+- se il libro non è ancora finito, e ci stanno ancora lavorando su, aggiungi "in process", come descritto [qui sotto.](#in_process)
+- se è richiesto un indirizzo email o un account per poter scaricare il libro, aggiungilo tra parentesi, esempio: `(email address *requested*, not required)`
+
+### Formattazione
```
Comment (davorpa) [inline CONTRIBUTING-it.md:50]: :warning: Marker anchor at line 21 doesn't work. Please, apply :wink: this snippet

```markdown
<a name="formatting"></a>
### Formattazione
```

File: HOWTO-it.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+* [Youtube - Github Tutorial per Principianti](https://www.youtube.com/watch?v=0fKg7e37bQE)
+* [Youtube - Come forkare una Repository Github e Inviare una Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw)
+* [Youtube - Corso accelerato di Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo)
+
+
+Non esitare a fare domande; ogni collaboratore iniziato con una prima pull request (PR). Potresti essere il nostro millesimo collaboratore.
+
+Anche se sei un collaboratore esperto in ambito open source, queste sono cose che potrebbero crearti dei problemi. Una volta che hai inviato la tua PR, Github Actions  avvierà un linter, spesso trovando piccoli problemi di spaziatura o di ordinamento alfabetico. Se ottieni una spunta verde, tutto è pronto per una revisione, ma se non è così clicca su "Details" sotto il check che è stato fallito, analizza l'errore, risolvilo e aggiungi un commit alla PR. 
+
+In fine, se non sei sicuro che la risorsa che vuoi aggiungere è appropiata a Free-Programming-Books, leggi le linee guida su [CONTRIBUTING](CONTRIBUTING.md) (inglese).
```
Comment (davorpa) [inline HOWTO-it.md:17]: Link CONTRIBUTING with the localized resource that you also provide :stuck_out_tongue_winking_eye: 

```markdown
In fine, se non sei sicuro che la risorsa che vuoi aggiungere è appropiata a Free-Programming-Books, leggi le linee guida su [CONTRIBUTING](CONTRIBUTING-it.md).
```

File: HOWTO-it.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+Leggilo in altre lingue: [English](HOWTO.md), [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [Italiano](HOWTO-it.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md), [Русский](HOWTO-ru.md)
```
Comment (davorpa) [inline HOWTO-it.md:1]: :information_source: Maybe link also the other translation not merged yet?

- #5684 German (at this moment conficted with Russian)

File: CONTRIBUTING-it.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
+*Leggilo in altre lingue: [English](CONTRIBUTING.md), [Français](CONTRIBUTING-fr.md), [Español](CONTRIBUTING-es.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md).*
```
Comment (davorpa) [inline CONTRIBUTING-it.md:1]: :information_source: An oportunity to link other translations not merged yet :tada: 

- #5720 German

**Review by davorpa — CHANGES_REQUESTED** [review]
:information_source: Some minor bugs

Be sure that review suggestions are applied :wink:

File: CONTRIBUTING-it.md [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+
+
+##### Tutorial interattivi vs. Altro
+
+Se riesci a stamparlo e conservarne l'essenza, non è un tutorial interattivo.
+
+
+### Automazione
+
+- L'applicazione delle regole di formattazione è automatizzata tramite [GitHub Actions](https://github.com/features/actions) usando [fpb-lint](https://github.com/vhf/free-programming-books-lint) (guarda [.github/workflows/fpb-lint.yml](.github/workflows/fpb-lint.yml))
```
Comment (davorpa) [inline CONTRIBUTING-it.md:207]: It satisfy #5644

File: CONTRIBUTING-it.md [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
+
+    - *Libri* : PDF, HTML, ePub, gitbook.io, una Git repo, etc.
+    - *Corsi* : Un corso è del materiale gratuito che non è un libro. [Questo è un corso](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-006-introduction-to-algorithms-fall-2011/).
+    - *Tutorial Interattivi* : Un sito interattivo permette all'utente di scrivere codice o comandi e analizzarne il risultato. esempi: [Try Haskell](http://tryhaskell.org), [Try Github](http://try.github.io).
+    - *Podcasts e Screencasts* : Podcasts and screencasts.
+    - *Set di problemi & Programmazione competitiva* : Un sito o software che ti permette di valutare le tue skills da programmatore risolvendo problemi semplici o complessi, con o senza la revisione del codice, con o senza la comparazione del risultato con gli altri utenti.
+
+4. Assicurati di seguire le [linee guida qui sotto](#guidelines) e rispettare la [formattazione Markdown](#formatting) dei file.
+
+5. GitHub Actions avvierà dei test per assicurarsi che le tue liste siano ordinate alfabeticamente e formattate correttamente. Assicurati che i tuoi cambiamenti passino il test.
```
Comment (davorpa) [inline CONTRIBUTING-it.md:23]: It satisfy #5644

**Comment by LuigiImVector** [comment]
Thank you so much @davorpa, I fixed everything.

File: CONTRIBUTING-it.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-*Leggilo in altre lingue: [English](CONTRIBUTING.md), [Français](CONTRIBUTING-fr.md), [Español](CONTRIBUTING-es.md), [Deutsch](CONTRIBUTING-de.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md).*
+*Leggilo in altre lingue: [Deutsch](CONTRIBUTING-de.md), [English](CONTRIBUTING.md), [Español](CONTRIBUTING-es.md), [Français](CONTRIBUTING-fr.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md).*
```
Comment (davorpa) [inline CONTRIBUTING-it.md:1]: Ok at this moment.

I have to remember alphabetize lists when all translation will be stable

**Review by davorpa — APPROVED** [review]
Thanks for this big effort ^-^,

### Ready to merge :fireworks: :tada:

**Comment by eshellman** [comment]
sorry about the conflicts, we're merging older PRs first

**Comment by LuigiImVector** [comment]
> 
> 
> sorry about the conflicts, we're merging older PRs first

What should I do now? Do I wait for older PRs to be mergied and then resolve conflicts?

**Comment by eshellman** [comment]
I would separate the translation and the links to it into two separate PRs. The links generate the conflicts, and we can do a cleanup PR after things have settled a bit.

**Comment by LuigiImVector** [comment]
so... I remove all links and then fix everything in the future? I have not understood well

**Comment by eshellman** [comment]
The crosslinks in the other language files are causing the conflicts. don't bother updating the other files, that will remove the conflicts

**Comment by LuigiImVector** [comment]
> 
> 
> The crosslinks in the other language files are causing the conflicts. don't bother updating the other files, that will remove the conflicts

Oh right, i'm so dumb. Now everything should be fine.

**Comment by davorpa** [comment]
@LuigiImVector you was the latest... you win the hell merge lotery 😂

File: CODE_OF_CONDUCT-it.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
+# Codice di Condotta dei collaboratori
```
Comment (davorpa) [inline CODE_OF_CONDUCT-it.md:1]: Take a look to this:

https://www.contributor-covenant.org/it/version/1/3/0/code-of-conduct/code_of_conduct.md

Comment (LuigiImVector) [reply CODE_OF_CONDUCT-it.md:1]: They are pretty similar, do you want that I change my translation with the official one?

File: CONTRIBUTING-it.md [hunk 11]
```diff
@@ -0,0 +1,218 @@
...
```
Comment (davorpa) [inline CONTRIBUTING-it.md:1]:
```suggestion
*Leggilo in altre lingue: [Deutsch](CONTRIBUTING-de.md), [English](CONTRIBUTING.md), [Español](CONTRIBUTING-es.md), [Français](CONTRIBUTING-fr.md), **Italiano**, [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md).*
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT-it.md (hunk 10 above)
Comment (davorpa) [reply CODE_OF_CONDUCT-it.md:1]: Original is always the best option, if you are sure that meaning is the same

**Comment by LuigiImVector** [comment]
Thanks  @davorpa for your suggestions

File: CODE_OF_CONDUCT-it.md (hunk 10 above)
Comment (LuigiImVector) [reply CODE_OF_CONDUCT-it.md]: Done, thanks for the help

File: CONTRIBUTING-it.md [hunk 12]
```diff
@@ -3,8 +3,8 @@
 ## Accordo di Licenza
 Contribuendo tu accetti alla [LICENZA](https://github.com/EbookFoundation/free-programming-books/blob/master/LICENSE) di questa repository.
 
-## Codice di Condotta dei Collaboratori
-I collaboratori accettano di rispettare il [Codice di Condotta](https://github.com/EbookFoundation/free-programming-books/blob/master/CODE_OF_CONDUCT-it.md) di questa repository.
+## Codice di Comportamento del Collaboratore
```
Comment (davorpa) [inline CONTRIBUTING-it.md:6]: Wow! syncing resources!! Thanks for taking care

**Review by davorpa — APPROVED** [review]
WOW. 58 commits is worthy of mention for guiness record awards. A nice trip reviewing too.

**Comment by eshellman** [comment]
yaay

**Pull request merged**